### PR TITLE
WIP DPL067  PBMC Pooling Plate transfer

### PR DESCRIPTION
**Pull request by harrietc52**
WIP

TODO: 
- Presenter class
- Incorporate CSV story?

### CODE REVIEW

File: app/models/labware_creators/cardinal_pools_plate.rb [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+        pool.each do |well|
+          source_position = well.location
+          result[source_position] = { dest_locn: destination_well_location }
+        end
+      end
+      result
+    end
+
+    def build_pools
+      pools = [[], [], [], [], [], [], [], []]
```
Comment (BenTopping) [inline app/models/labware_creators/cardinal_pools_plate.rb:102]: needs to be be changed to be number_of_pools in length

Comment (BenTopping) [reply app/models/labware_creators/cardinal_pools_plate.rb:102]: might be able to change back to original, within line 107 loop, where if pool[current_pool] doesnt exist, then create it.

File: config/cardinal_pooling.csv [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+30,10,10,10,,,,,
+29,10,10,9,,,,,
+28,10,9,9,,,,,
+27,9,9,9,,,,,
+26,13,13,,,,,,
+25,13,12,,,,,,
+24,12,12,,,,,,
+23,12,11,,,,,,
+22,11,11,,,,,,
+21,10,11,,,,,,
```
Comment (sdjmchattie) [inline config/cardinal_pooling.csv:77]: Is this one right?  It's the only one I can see where the numbers are not in equal-or-descending order.

**Comment by harrietc52** [comment]
Closing as https://github.com/sanger/limber/pull/833/files is replacing it

**Comment by harrietc52** [comment]
Closing as https://github.com/sanger/limber/pull/833/files is replacing it

**Pull request closed without merging**